app/components/featured_image_component.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class FeaturedImageComponent < ViewComponent::Base
+  delegate :cms_image, to: :helpers
```

Comment:
cms_image appears to be unused here?

Reply:
Good spot, missed that when I refactored this